extensions/job_management/standard/requirements/openeo/update/REQ_body.adoc

```diff
@@ -0,0 +1,7 @@
+[[req_openeo_update__body]]
+[requirement]
+====
+[%metadata]
+label:: /req/openeo/update-body
+part:: The media type `application/json` SHALL be used to indicate that request body contains a job encoded as an <<rc_openeo,OpenEO Process Graph>>.
```

Comment:
Same as above. If this is for PATCH /jobs/:id, a user must submit a (partial) job definition.
If this is for something else, a user would probably submit a UDP, not a process graph (generally, process graph is a term usually not used in the openEO API spec as a sole entity, instead UDP is used, which has a different schema).